Source/WebCore/dom/Element.cpp

```diff
@@ -2651,7 +2651,7 @@ void Element::invalidateForQueryContainerSizeChange()
 
 void Element::invalidateForAnchorRectChange()
 {
-    Node::invalidateStyle(Style::Validity::ElementInvalid);
+    Node::invalidateStyle(Style::Validity::ElementInvalid, Style::InvalidationMode::RebuildRenderer);
 }
```

Comment:
This would be very slow. We'd rebuild the render tree of the anchored element every time the anchor moves. Why is this needed?

Reply:
I think it should be sufficient to just invalidate layout for the renderer in case anchor-center.

Reply:
I was under the impression "invalidating layout" and "rebuild renderer" are the same thing, so that's why. I didn't realize the implication of rebuilding renderers.